### PR TITLE
W-11934107-APISharing-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 1]
```diff
@@ -297,23 +297,93 @@ To enable email notifications:
...
 To link Composer to Anypoint Platform:
 
-* Log in to Anypoint Platform.
-* In the navigation bar or the main Anypoint Platform page, click *Access Management*.
-* In the xref:access-management::managing-your-account.adoc[Access Management] navigation menu, click *Composer Sync*.
-* In the *Composer Sync Orgs* page, click *Add Composer Orgs*.
-* In the *Add organization* window, in the *Organization ID* box, enter your Composer organization and then click *Add*.
-+
-[NOTE]
+. In MuleSoft Composer, in the left pane, click *Settings* > *Account*. +
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:307]: "left" is not an accessible term. Is there another way to say this?

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 2]
```diff
@@ -297,23 +297,93 @@ To enable email notifications:
...
+== API Sharing
+
+MuleSoft Anypoint Platform customers can share externally available APIs managed in Anypoint Platform with business users for consumption in a MuleSoft Composer flow.
+
+Before sharing APIs from Anypoint Platform to MuleSoft Composer, ensure that:
+
+* Your organization uses Anypoint Platform for API Management.
+* Your organization has xref:exchange::to-create-an-asset.adoc#create-an-api-asset[Rest APIs] managed by API Manager that conform to the following:
+
+** API specifications use either RAML or OpenAPI
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:347]:
```suggestion
** API specifications use either RAML or OpenAPI.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 3]
```diff
@@ -297,23 +297,93 @@ To enable email notifications:
...
+** API uses basic authentication, bearer token, or API Key in conjunction with a Client ID and Secret
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:348]:
```suggestion
** API uses basic authentication, bearer token, or API Key in conjunction with a Client ID and Secret.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc (hunk 2 above)
Comment (dhirsch1) [reply en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:347]: Optional: I would also add "The" to the two list items: "The API specifications..." and "The API"...

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 4]
```diff
@@ -297,23 +297,93 @@ To enable email notifications:
...
+
+* Composer organizations are <<connect-composer-to-anypoint-platform,linked to Anypoint organizations>>.
+* API must be accessible from the public internet.
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:351]:
```suggestion
* The API must be accessible from the public internet.
```

**Review by dhirsch1 — DISMISSED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc (hunk 1 above)
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:307]: I will fix this!

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**